test/fixtures/server/filters.json

```diff
@@ -28,10 +28,21 @@
       "path": "/echo-query/:param?",
       "method": "GET",
       "origin": "http://localhost:${originPort}"
+    },
+
+    {
+      "path": "/test-blob/*",
+      "method": "GET",
+      "origin": "http://localhost:${originPort}",
     }
 
   ],
   "public": [
+    {
+      "path": "/test-blob/*",
+      "method": "GET",
+      "stream": true
```

Comment:
Changing this flag will fail the tests as the response will be encoded by the non-streaming code.